android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/component/TopBar.kt

```diff
@@ -274,6 +286,7 @@ fun MullvadTopBarWithDeviceName(
             onSettingsClicked,
             onAccountClicked,
             Modifier,
+            enabled = true,
```

Comment:
I was thinking that we can expose it when we need it, because right now we would only expose it but never use it on the other top bars. I'm open to adding it if we feel it make sense.

Reply:
I think it's good enough for this change, so will resolve this.